src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsstrengthsbasedneedsassessmentsapi/formconfig/FormConfigProvider.kt

```diff
@@ -44,8 +44,8 @@ class FormConfigProvider(
       throw FormConfigNotFoundException("Unable to fetch form config from ${request.uri()}")
     }
 
-    val decoder = Json { ignoreUnknownKeys = true }
```

Comment:
With the class using the `@Componenent` annotation, adding the `objectMapper` to the constructor args similar to the `client` may allow for it to be auto-wired rather than creating a new instance here